numpy/core/src/umath/loops.c.src

```diff
@@ -19,6 +19,7 @@
 #include "ufunc_object.h"
 
 #include <string.h> /* for memchr */
+#include <libdivide.h>
```

Comment:
How did you add libdivide.h to the PR?

Reply:
This is more of a local change, which is why the [build](https://travis-ci.com/github/numpy/numpy/jobs/430635092#L4664-L4665) is failing. Will look at zlib, thanks.

Reply:
I think you meant to add the file but it does not appear in the PR

Reply:
Oh, yes I wanted to add it. I will add the standard repo version of the header now to check if builds and UT passes. Meanwhile, I will explore the zlib version and modifications to SIMD. Thanks for the info.